lib/inspec/config.rb

```diff
@@ -328,21 +328,35 @@ def validate_config_file_contents!
     def validate_reporters!(reporters)
       return if reporters.nil?
 
-      # TODO: move this into a reporter plugin type system
-      valid_types = %w{
-        automate
-        cli
+      # These "reporters" are actually RSpec Formatters.
+      # json-rspec is our alias for RSpec's json formatter.
+      rspec_built_in_formatters = %w{
         documentation
         html
+        json-rspec
+        progress
+      }
+
+      # These are true reporters, but have not been migrated to be plugins yet.
+      unmigrated_reporters = %w{
```

Comment:
In the spirit of naming things to reflect current implementation, this could be named `inspec_reporters`.
Naming these `unmigrated`  might suggest that migration is a thing that happens somewhere inside the current InSpec code. 
The comment on the line above leaves a record that these need migrated at some point (we should make sure there's a ticket for doing this as well).